lib/board.rb

```diff
@@ -0,0 +1,83 @@
+require_relative './game.rb'
+
+class Board < Game
+  def initialize
+    @game = Game.new(@players)
+  end
+
+  def welcome_message
+    puts 'Welcome to Tic Tac Toe ! ! !'
+  end
```

Comment:
- [ ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- This message should be printed in the main.rb file